TrkDiag/src/ComboHitDiag_module.cc

```diff
@@ -41,55 +40,71 @@ namespace mu2e
 {
   class ComboHitDiag : public art::EDAnalyzer {
     public:
-      explicit ComboHitDiag(fhicl::ParameterSet const&);
+
```

Comment:
Here too, a bit of formatting make reading the code much easier.... Yes yes, I know.